### PR TITLE
REF remove extra base_version key to allow bot to update python

### PR DESCRIPTION
**Pull request by beckermr**
Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

This change will allow the auto-tick bot to push version updates I think.

### CODE REVIEW

**Comment by beckermr** [comment]
@conda-forge-admin rerender

**Comment by beckermr** [comment]
We are missing this one on drone

```

conda.exceptions.ResolvePackageNotFound:
--
471 | - libx11-devel-cos7-aarch64

```

**Comment by isuruf** [comment]
@beckermr, I restarted https://dev.azure.com/conda-forge/feedstock-builds/_build?definitionId=9061&_a=summary

**Comment by beckermr** [comment]
Looks like that job failed too with another error?

```
conda.exceptions.ResolvePackageNotFound: 
  - libx11-cos7-aarch64==1.6.7
```

**Comment by beckermr** [comment]
@conda-forge-admin, please rerender

**Comment by isuruf** [comment]
Drone build succeeded, but it's shown as pending still. Ready for review

**Comment by mariusvniekerk** [comment]
Looks fine to me

File: recipe/meta.yaml [hunk 1]
```diff
@@ -101,8 +100,6 @@ build:
   skip: True            # [win and int(float(vc)) < 14]
 # Would like to be able to append to the calculated build string here, ping @msarahan, is this possible?
   string: h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}{{ linkage_nature }}{{ debug }}_cpython
-  run_exports:
-   - python_abi 3.8.* cp38
```
Comment (scopatz) [inline recipe/meta.yaml]: Is it correct to remove run_exports @conda-forge/python?

Comment (isuruf) [reply recipe/meta.yaml]: This needs to be added after there's a new conda-build release. There's also https://github.com/conda-forge/python-feedstock/pull/310 which uses the correct syntax for the new conda-build release.

Comment (beckermr) [reply recipe/meta.yaml]: @isuruf

Comment (scopatz) [reply recipe/meta.yaml]: Alright then!

**Pull request merged**